File: sample_recipe/178dm.py
# !/usr/bin/env python
# -*- coding:utf-8 -*-

from feed2_base import Feed

class Dm178(Feed):
    name = '178dm'
    oldest = 1
    capture = {}
    capture["block_img"] = ['159502145898']
    url = r'http://acg.178.com/s/rss.xml'

recipe = Dm178

File: sample/178dm.js

const Base = require('../common/recipe/feed.js')

module.exports = class Feed extends Base{
  init(){
    super.init()
    this.name = '178动漫'
    this.url = 'http://acg.178.com/s/rss.xml'
    this.capture.blockImg = ['159502145898']
    this.capture.oldest = 1
  }
}
